lib/cs_guide_web/templates/components/drink_card.html.eex

```diff
@@ -41,4 +42,5 @@
       <% end %>
     </div>
   </div>
+  </label>
```

Comment:
yes, this allow us to check/uncheck the checkbox which hide/show the drink card details with css.
The label nest all the card to allwo user to click anywhere on the drink card to toggle the "more details" section

Reply:
@SimonLab understood. This _should_ work in most modern browsers, but it's not the HTML5 spec (_which expressly forbids any nested elements in a `<label`> element._) 👍

Reply:
:+1: 
my thought was to avoid using javascript for the html drink card and to avoid adding more msg/logic on the elm drink card and to try to continue to use a css solution

Reply:
@SimonLab I **100% _agree_** with a CSS solution. (_always better than JS!_) 
My only question is: can we use a _different_ HTML element to avoid _nesting_ `<label>`s?

Reply:
From my previous research, I think it's not possible to use another html tag other than label to active the input checkbox. I'm going to have a better look to see if there is another solution (without js)

Reply:
If after 1 x 🍅of research, you are unable to find a _better_ way of activating a checkbox without  nesting `<label>`s, please open an issue in https://github.com/dwyl/learn-html5/issues describing the use-case & challenge you are facing. Link to it from here and re-assign the PR to me. 👍

Reply:
No luck so far. I've created an issue on learn-html5 (don't have the right to add labels on the issue) https://github.com/dwyl/learn-html5/issues/43

Should I add some javascript to remove the label?

Reply:
@SimonLab I would _prefer_ to avoid JS as CSS is more resilient and easier to maintain. 
Thanks for opening the issue.
Please assign PR back to me and let's :shipit: